src/Game/GameActions.cs

```diff
@@ -324,8 +324,8 @@ public static bool PickUp
 
                     gump = new SplitMenuGump(item, new Point(x, y))
                     {
-                        X = Mouse.LClickPosition.X - 80,
-                        Y = Mouse.LClickPosition.Y - 40
+                        X = Mouse.Position.X - 80,
```

Comment:
This has been fixed time ago because it caused lots of issues when fast dragging

Reply:
I just tried to pick up a stacked item while my mouse was moving as fast as possible and I can't get any problem to happen no matter how fast I go. This change should make the gump always appear under where your mouse currently is, as opposed to where you clicked initially. Most of the time that's the same place, but if your mouse is moving very quickly it can shift a few pixels off, and I think this behavior is better.

Reply:
![Vbl78RdYDw](https://user-images.githubusercontent.com/20810422/94972715-3ad85500-050a-11eb-8048-07816544eeb1.gif)

just noticed this with sallos easy grab enabled

Reply:
ok I can reproduce this. I'll get to the bottom of it.

Reply:
ok, updated the PR with this fixed. I actually reworked a bit how control dragging works and I think it's much simpler than before. I don't think previously the _dragOffset stuff really worked with easy-grab (it would set it to wherever you started dragging, which may be very far away), and there was some weird offset math with a few of the gumps.